### ISSUE TITLE
I can't find what I'm looking for edit only certain areas

### ISSUE THREAD

**Issue by Remy-Limelight** [tinymce/tinymce-docs#1376]
I am building a mailer app

I have a div that  Iuse to enable the editor, but everything inside that div is editable

How can I exclude divs, table, tr etc. and only edit the p, span, h1, h2 etc. that are inside the tr's?

**Comment by tylerkelly13** [comment]
There is two strategies you could use here:
1. Use the Noneditable plugin and add the `noneditable_noneditable_class` class to the non-editable content: https://www.tiny.cloud/docs/plugins/noneditable/

**Comment by tylerkelly13** [comment]
There is two strategies you could use here:
1. Use multiple inline editors: https://www.tiny.cloud/docs/general-configuration-guide/use-tinymce-inline/
2. Use the Noneditable plugin and add the `noneditable_noneditable_class` and `noneditable_editable_class` class to the non-editable content: https://www.tiny.cloud/docs/plugins/noneditable/

**Comment by Remy-Limelight** [comment]
I tried the Noneditable plugin, but it makes the editor act weird

my setup:
div // this is where i init the editor // ignore completely 
    table // ignore completely 
        tr // ignore completely 
            td // ignore completely 
                h1 // i only want to edit this
                p // i only want to edit this

when i add the noneditable classes to the tr and td then when i click on them they dissapear... i am creating a mailer building app

in short, the tiny api works a little too well because it want to edit every single thing

**Comment by tylerkelly13** [comment]
Please note that the issues section of the TinyMCE documentation repository is for reporting documentation issues.
If you are having trouble using TinyMCE or have questions about a specific use-case, please visit [TinyMCE questions on Stack Overflow](https://stackoverflow.com/questions/tagged/tinymce) or the [TinyMCE Community](https://community.tiny.cloud/).
